botbot/Module/OpenAIModule.cs

```diff
@@ -24,26 +25,49 @@ public OpenAIModule()
             encoder = Encoder.Get_Encoder();
         }
 
+        private string GetPrefix(string text) {
+            string[] splitText = text.Split(' ');
+            string command = splitText[1];
+            if (String.Equals(command, "eli5") ){
+                return "Explain like I'm 5 years old: ";
+            }
+
+            return "";
+        }
+
+        private string GetPrompt(string text) {
+            string[] splitText = text.Split(' ');
+            if (splitText[2].StartsWith("temp")) {
+                return text.Substring(
+                    splitText[0].Length + splitText[1].Length + splitText[2].Length + 2
+                ).Trim();
+            }
+            return text.Substring(splitText[0].Length + splitText[1].Length + 1).Trim();
+        }
+
+        private float GetTemp(string text) {
+            string[] splitText = text.Split(' ');
+            if (splitText[2].StartsWith("temp")) {
+                string[] splitTemp = splitText[2].Split(':');
+                return float.Parse(splitTemp[1]);
+            }
+            return 1;
+        }
+
         public async Task<ModuleResponse> Handle(string text, string userId, string channel)
         {
-            if (text.ToLower().StartsWith("botbot gpt"))
+            if (text.ToLower().StartsWith("botbot gpt") || text.ToLower().StartsWith("botbot eli5"))
             {
-                text = text.Substring(10).Trim();
-                float temp = 1;
-                if (text.StartsWith("temp"))
-                {
-                    string[] splitText = text.Split(' ');
-                    string[] splitTemp = splitText[0].Split(':');
-                    temp = float.Parse(splitTemp[1]);
-                    text = text.Substring(splitText[0].Length).Trim();
-                }
+                string prefix = GetPrefix(text);
+                string prompt = prefix + GetPrompt(text);
```

Comment:
Don't really like the double string.Split here. Should split the text before and pass it to both methods.

Reply:
Sure, I ended up reducing to a single helper because it gets messy trying to work with the partial strings in the other helper methods